resources/utilities/upload_vcf.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
+
+"""
+Usage: $0 PATIENT_ID VCF CREDENTIALS_FILE SERVER_NAME
+
+Upload the VCF file as an attachment for the given patient. For example:
+
+    $0 P0000102 path/to/file.vcf path/to/credentials.txt server_name
```

Comment:
1. There's no mention of the port, but this actually supports specifying a port. Use two examples:

    $0 P0000102 path/to/file.vcf path/to/credentials.txt phenomecentral.org
    $0 P0000102 path/to/file.vcf path/to/credentials.txt localhost:8080

2. This doesn't work when PhenoTips is not deployed as the root webapp. For example, in the standalone distribution edit `jetty/contexts/phenotips.xml` and change to: `<Set name="contextPath">/pt</Set>`. Either add another optional parameter for the context path, or support something like `localhost:8080/pt`.